src/signing/interactors/serial_batch_signing_request.rs

```diff
@@ -0,0 +1,26 @@
+use crate::prelude::*;
+
+/// A batch signing request used with a SignWithFactorSerialInteractor, containing
+/// a collection of transactions to sign with multiple keys (derivation paths),
+/// and a collection of transactions which would be invalid if the user skips
+/// signing with this factor source.
+#[derive(derive_more::Debug)]
+#[debug("input: {:#?}", input)]
+pub struct SerialBatchSigningRequest {
+    pub input: BatchTXBatchKeySigningRequest,
+    /// A collection of transactions which would be invalid if the user skips
+    /// signing with this factor source.
+    pub invalid_transactions_if_skipped: Vec<InvalidTransactionIfSkipped>,
```

Comment:
Is this meant to be used by the hosts to know if a given signing request can be skipped?
If yes, then couldn't this be just a flag telling if signing with a given factor can be skipped or not? I mean that at the UI level we would show only one factor source access slide up.

Reply:
see https://github.com/radixdlt/sargon-mfa/pull/1#discussion_r1734190284

Matt wanted to be able to provide more context than just a boolean flag. The UI would be updated with a new button "Skip".